package/tessel/tessel-mdns/Makefile

```diff
@@ -0,0 +1,24 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tessel-mdns
+PKG_VERSION:=0.1
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/tessel-mdns
+	CATEGORY:=Tessel
+	TITLE:=Begin advertising Tessel over mDNS
+endef
+
+define Build/Compile
```

Comment:
@kevinmehall I'm going to go ahead and remove this section as well unless you advise otherwise.